src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -50,15 +50,18 @@
        :progress-bar-width progress-bar-width}]]))
 
 (defn f-view
-  [{:keys [animate? progress paused? header-text background header-background swipeable?]}]
+  [{:keys [animate? progress paused? header-text background header-background gesture]}]
```

Comment:
Changed the carousel API.

Since the carousel cannot be  `tappable?` and `swipable?` at the same time, I added a `gesture` key that receives `:tappable` or `:swipeable` keywords